### PR TITLE
i2pd: update to 2.38.0

### PR DESCRIPTION
**Pull request by r4sas**
Signed-off-by: R4SAS <r4sas@i2pmail.org>

Topic Description
-----------------
i2pd: update to 2.38.0

Package(s) Affected
-------------------
i2pd

Security Update?
----------------
No

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-web/i2pd/autobuild/defines [hunk 1]
```diff
@@ -4,9 +4,7 @@ PKGDEP="boost miniupnpc openssl"
 PKGDES="End-to-End encrypted and anonymous Internet"
 
 ABSHADOW=0
-CMAKE_AFTER="-DWITH_AESNI=OFF \
-             -DWITH_AVX=OFF \
```
Comment (MingcongBai) [inline extra-web/i2pd/autobuild/defines:8]: We default to SSE2 target on AMD64, which is why we have disabled AES-NI and AVX.

Comment (MingcongBai) [reply extra-web/i2pd/autobuild/defines:8]: Please let me know if I'm incorrect.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-web/i2pd/autobuild/defines (hunk 1 above)
Comment (r4sas) [reply extra-web/i2pd/autobuild/defines:8]: At 2.35.0 (+ some fixes was done after release, so fully changed at 2.36.0) I've reworked part of AVX and AESNI detection, so it must work without problems.
See our tracker: https://github.com/PurpleI2P/i2pd/issues/1600 and https://github.com/PurpleI2P/i2pd/issues/1602

**Review by liushuyu — APPROVED** [review]

File: extra-web/i2pd/autobuild/defines (hunk 1 above)
Comment (MingcongBai) [reply extra-web/i2pd/autobuild/defines:8]: Understood.

**Review by MingcongBai — APPROVED** [review]

**Comment by MingcongBai** [comment]
Some patches, according to @liushuyu, could not be applied against v2.38.0. A rework pull request will be created shortly.

**Pull request merged**